docs/Salesforce-batchsink.md

```diff
@@ -28,6 +28,8 @@ A Salesforce connected application must be created in order to get a client secr
 
 **Login Url:** Salesforce OAuth2 login url.
 
+**Connect Timeout:** Maximum time in milliseconds connection initialization can take.
```

Comment:
Change to "Maximum time in milliseconds to wait for connection initialization before time out." in all docs.

Reply:
done